dingo/core/utils/torchutils.py

```diff
@@ -4,6 +4,13 @@
 from torch.nn import functional as F
 from torch.utils.data import DataLoader
 from typing import Union, Tuple, Iterable
+import bilby
+
+
+def fix_random_seeds(_):
+    """Util function to set random seeds when using multiple workers for Dataloader."""
+    np.random.seed(int(torch.initial_seed()) % (2 ** 32 - 1))
+    bilby.core.utils.random.seed(int(torch.initial_seed()) % (2 ** 32 - 1))
```

Comment:
This should probably be placed in a `try`/`except` statement in case an older version of Bilby is used.